server/db/db.js

```diff
@@ -1,5 +1,15 @@
 const db = require('./builder');
```

Comment:
@AnnaC91 let's change the name of the /db/db.js file to index.js  so we can simply require in all of our models as './db' 

then change the file of `builder.js` to `_db.js` as it is common convention to indicate that this is an internal file only